controllers/storagecluster/exporter.go

```diff
@@ -709,3 +715,86 @@ func createMetricsExporterRolebindings(ctx context.Context, r *StorageClusterRec
 
 	return nil
 }
+
+func createRookCephClusterRolebindings(ctx context.Context,
+	r *StorageClusterReconciler, instance *ocsv1.StorageCluster) error {
+	// rookCephMonitorMgrRoleBinding is a cluster rolebinding for monitor mgr
+	rookCephMonitorMgrRoleBinding := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rook-ceph-monitor-mgr",
+			Namespace: instance.Namespace,
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "rook-ceph-monitor-mgr",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind: "ServiceAccount",
+				Name: "rook-ceph-mgr",
+			},
+		},
+	}
+
+	// rookCephMonitorRoleBinding is a cluster rolebinding for rook-ceph monitor
+	rookCephMonitorRoleBinding := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rook-ceph-monitor",
+			Namespace: instance.Namespace,
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "rook-ceph-monitor",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind: "ServiceAccount",
+				Name: "rook-ceph-system",
+			},
+		},
+	}
+
+	var roleBindings = []*rbacv1.ClusterRoleBinding{
+		rookCephMonitorMgrRoleBinding, rookCephMonitorRoleBinding,
+	}
+
+	for _, expectedRoleBinding := range roleBindings {
+		currentClusterRoleBinding := new(rbacv1.ClusterRoleBinding)
+		currentClusterRoleBinding.Name = expectedRoleBinding.Name
+		currentClusterRoleBinding.Namespace = instance.Namespace
+
+		_, err := controllerutil.CreateOrUpdate(ctx, r.Client, currentClusterRoleBinding, func() error {
+			// add expected role reference
+			if currentClusterRoleBinding.CreationTimestamp.IsZero() {
+				// RoleRef is immutable. So inject it only while creating new object.
+				currentClusterRoleBinding.RoleRef = expectedRoleBinding.RoleRef
+			}
+
+			// add expected subjects
+			expectedSubjectsLen := len(expectedRoleBinding.Subjects)
+			currentClusterRoleBinding.Subjects = make([]rbacv1.Subject, expectedSubjectsLen)
+			copy(currentClusterRoleBinding.Subjects, expectedRoleBinding.Subjects)
+
+			// add owner references
+			currentClusterRoleBinding.OwnerReferences = []metav1.OwnerReference{{
+				APIVersion: instance.APIVersion,
+				Kind:       instance.Kind,
+				Name:       instance.Name,
+				UID:        instance.UID,
+			}}
```

Comment:
ClusterRoleBindings should not have OwnerReferences.

Reply:
Done